FXyz-Importers/build.gradle

```diff
@@ -2,4 +2,5 @@ ext.moduleName = 'org.fxyz3d.importers'
 
 dependencies {
     api (project(':FXyz-Core')) { transitive = false }
-}
\ No newline at end of file
+    implementation (project(':FXyz-Core'))
```

Comment:
Having both api and implementation for the core project doesn't make sense, imo.

Would this work:
```
dependencies {
    api (project(':FXyz-Core')) { transitive = false }
    runtimeOnly 'org.slf4j:slf4j-api:2.0.7'
}
```
or:
```
dependencies {
    api (project(':FXyz-Core')) { transitive = false }
    implementation 'org.slf4j:slf4j-api:2.0.7'
}
```
?

Reply:
Yes, I was trying to re-use the `project` to avoid the overhead to keep the versions in sync in two places.

You can also use this to keep the the GMM exactly the same for runtimeElements as well.

```
dependencies {
    api (project(':FXyz-Core')) { transitive = false }
    runtimeOnly (project(':FXyz-Core'))
}
```

Reply:
This is also a possibility since slf4j is only needed in the importers tests

````grooby
dependencies {
    api (project(':FXyz-Core')) { transitive = false }
    testRuntimeOnly (project(':FXyz-Core'))
}

Reply:
Yes, that's better.

In any case, just for using one dependency of the whole core project, I'm still in favour of including only such dependency, and look for a better way to find its version (for instance with a variable in the parent gradle file that can be reused by core and imports).

Reply:
Ah I see. If you support using properties here, I'll do so :)

Reply:
See 910e8807a44afdcf23eda154108b9162ecb2e9e4